crates/sui-graphql-rpc/examples/object/object.graphql

```diff
@@ -7,11 +7,25 @@
     digest
     storageRebate
     owner {
-      defaultNameServiceName
```

Comment:
this would be under AddressOwner { owner } now right?

Reply:
Yes, these examples + tests need some love now.